docs/getting-started-guides/vsphere.md

```diff
@@ -55,6 +55,31 @@ export GOVC_INSECURE=1
 govc vm.change -e="disk.enableUUID=1" -vm=<VMNAME>
 ```
 
+* Create Role and User with Required Privileges for vSPhere Cloud Provider
+
+vSphere Cloud Provider requires following minimal set of privileges to interact with vCenter.
```

Comment:
"requires following" -> "requires the following"
Replace period at the end of the sentence with colon ":"